app/src/main/res/layout/item_bill.xml

```diff
@@ -37,65 +37,91 @@
 
             <TextView
                 android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
+                android:layout_height="match_parent"
                 android:background="@color/colorPrimary"
+                android:gravity="center"
+                android:minWidth="@dimen/dp_20"
                 android:padding="@dimen/dp_2"
                 android:text="@{position + 1 + ``}"
                 android:textColor="@color/color_white"
                 android:textSize="@dimen/sp_18"/>
 
             <LinearLayout
                 android:layout_width="0dp"
-                android:layout_height="wrap_content"
-                android:layout_marginEnd="@dimen/dp_10"
-                android:layout_marginRight="@dimen/dp_10"
+                android:layout_height="match_parent"
                 android:layout_weight="1"
-                android:background="@color/color_white"
-                android:orientation="horizontal"
-                android:paddingLeft="@dimen/dp_5"
-                android:paddingRight="@dimen/dp_5"
-                android:paddingStart="@dimen/dp_5">
+                android:orientation="vertical"
+                android:paddingLeft="@dimen/dp_3"
+                android:paddingStart="@dimen/dp_3">
```

Comment:
WARNING: [RtlSymmetry] When you define `paddingStart` you should probably also define `paddingEnd` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.